MyCryptoHQ/MyCrypto
# [IN COMMUNITY RELEASE] RIF Token for mainnet and testnet

Closes #2273

### Description

Contract and Token for `RIF` in mainnet and testnet `RSK` network

### Changes

* Config files

### Steps to Test

1. Choose `RSK` network (either mainnet or testnet)
2. Go to "Contracts", then "Interact"
3. Check `RIF` contract is available for mainnet (or `tRIF` for testnet)
